src/test/java/org/broadinstitute/hellbender/utils/pairhmm/VectorPairHMMUnitTest.java

```diff
@@ -23,22 +24,27 @@
 
     private static final String pairHMMTestData = publicTestDir + "pairhmm-testdata.txt";
 
-   @BeforeClass
-    public void initialize() {
-       try {
-           new VectorLoglessPairHMM(null);
-       } catch (final Exception e) {
-           throw new SkipException("AVX PairHMM is not supported on this system or the library is not available");
-       }
-    }
-
+    // Return a list of supported VectorLoglessPairHMM implementations, skip the test if none are supported
     private List<N2MemoryPairHMM> getHMMs() {
-        final PairHMMNativeArguments args = new PairHMMNativeArguments();
+        List<N2MemoryPairHMM> list = new ArrayList<>();
+        PairHMMNativeArguments args = new PairHMMNativeArguments();
         args.useDoublePrecision = false;
         args.maxNumberOfThreads = 1;
-        final N2MemoryPairHMM avxPairHMM = new VectorLoglessPairHMM(args);
-        avxPairHMM.doNotUseTristateCorrection();
-        return Collections.singletonList(avxPairHMM);
+
+        for (VectorLoglessPairHMM.Implementation imp : VectorLoglessPairHMM.Implementation.values()) {
+            try {
+                final N2MemoryPairHMM avxPairHMM = new VectorLoglessPairHMM(imp, args);
+                avxPairHMM.doNotUseTristateCorrection();
```

Comment:
According to [this line](https://github.com/broadinstitute/gatk/blob/47d5416fe2590d02a3223d0e00c6d8a8b4e2b3d0/src/main/java/org/broadinstitute/hellbender/utils/pairhmm/PairHMM.java#L92), `doNotUseTristateCorrection` is for debugging purposes. By default, we use the tristate correction. To disable the tristate correction, code must call `doNotUseTristateCorrection()`. The only code calling `doNotUseTristateCorrection()` is in `VectorPairHMMUnitTest` and `PairHMMUnitTest`. git-blame says Adam and @lbergelson added these lines of code. I would say these lines are not needed, since adding these lines means the test behaves differently than the default tool usage. 

BTW, AVX PairHMM in GKL includes the tristate correction and does not have an option to disable it.